statchatexpress/models/userGame.js

```diff
@@ -0,0 +1,18 @@
+import { DataTypes } from "sequelize";
+import { sequelize } from "../database.js";
+
+export const UserGame = sequelize.define("UserGame", {
+  Gamertag: {
+    type: DataTypes.STRING,
+    allowNull: false,
+  },
+  Platform: {
+    type: DataTypes.STRING,
+    allowNull: false,
+  },
+  GameName: {
+    type: DataTypes.STRING,
+    allowNull: false,
+    unique: true,
```

Comment:
This should **not** be `unique`, or you're going to run into problems when a second user tries to save their profile. Please update this before merging. 

What does need to be unique is *every user/gameName/platform combination*. For example:

**OK**:
```
{ id: 1, UserId: 4, GameName: 'r4', Platform: 'PC' }
{ id: 1, UserId: 4, GameName: 'apex', Platform: 'PC' }
```
Not OK - should throw an error
```
{ id: 1, UserId: 4, GameName: 'r4', Platform: 'PC' }
{ id: 1, UserId: 4, GameName: 'r4', Platform: 'PC' }
```

Check out the `addConstraint` API to make sure this isn't allowed: https://stackoverflow.com/questions/34664853/sequelize-composite-unique-constraint

Reply:
just completed this